#57977 - Screen Elements UI Enhancement for Smaller Screens

xecurifyprashant opened the pull request:
Added a 2x2 grid for the screen options preferences for smaller screens.

[#57977](https://core.trac.wordpress.org/ticket/57977)

xecurifyprashant committed:
Merge remote-tracking branch 'upstream/trunk' into trunk

xecurifyprashant committed:
#57977

xecurifyprashant committed:
phpcs

xecurifyprashant committed:
Merge remote-tracking branch 'upstream/trunk' into trunk

xecurifyprashant committed:
Merge remote-tracking branch 'upstream/trunk' into trunk

xecurifyprashant committed:
Merge remote-tracking branch 'upstream/trunk' into trunk

xecurifyprashant committed:
Merge remote-tracking branch 'upstream/trunk' into trunk

xecurifyprashant committed:
Merge remote-tracking branch 'upstream/trunk' into trunk

mukeshpanchal27 commented on src/wp-admin/css/common.css:
```diff
@@ -3780,6 +3780,18 @@ img {
 		overflow-x: hidden;
 	}
 
+	.metabox-prefs-container
+	{
+		display: grid;
+		grid-template-columns: 1fr 1fr;
+	}
+
+	.metabox-prefs-container > *
+	{
+		display: inline-block;
+		padding: 5px;
+	}
+
```
```suggestion
	.metabox-prefs-container {
		display: grid;
		grid-template-columns: 1fr 1fr;
	}

	.metabox-prefs-container > * {
		display: inline-block;
		padding: 5px;
	}

```

mukeshpanchal27 commented on src/wp-admin/includes/class-wp-screen.php:
```diff
@@ -1120,6 +1120,8 @@ public function render_meta_boxes_preferences() {
 		</p>
 		<?php
 
+		echo ( '<div class="metabox-prefs-container">' );
+
```
```suggestion
		</p>
		<div class="metabox-prefs-container">
		<?php
```

xecurifyprashant committed:
Update src/wp-admin/css/common.css

Co-authored-by: Mukesh Panchal <mukeshpanchal27@users.noreply.github.com>

xecurifyprashant committed:
Update src/wp-admin/includes/class-wp-screen.php

Co-authored-by: Mukesh Panchal <mukeshpanchal27@users.noreply.github.com>

xecurifyprashant committed:
Merge remote-tracking branch 'origin/trunk' into trunk

xecurifyprashant committed:
Merge remote-tracking branch 'upstream/trunk' into trunk

xecurifyprashant committed:
Merge remote-tracking branch 'upstream/trunk' into trunk

xecurifyprashant committed:
UI Suggestions

xecurifyprashant committed:
Merge remote-tracking branch 'upstream/trunk' into trunk

xecurifyprashant committed:
#58407 

resetpassword action on users.php (users list page) handles retrieve_password() return incorrectly

xecurifyprashant committed:
Merge remote-tracking branch 'upstream/trunk' into trunk

audrasjb commented:
Closing in favor of #4674.
